### PR TITLE
Fix issues

### PR DESCRIPTION
**Pull request by pedrocarrico**
Fixed some of the pending issues.

@cwhetung let's test what goes to npm after we bump the version.

### CODE REVIEW

File: .npmignore [hunk 1]
```diff
@@ -0,0 +1,5 @@
+.nvmrc
+.travis.yml
+
+test/
+coverage/
```
Comment (cwhetung) [inline .npmignore]: we probably want to also add .git*

Comment (pedrocarrico) [inline .npmignore]: That's one of the already ignored things according to the [npm docs](https://docs.npmjs.com/misc/developers)

Comment (cwhetung) [inline .npmignore]: `.git` is ignored, but it doesn't look like stuff such as `.gitconfig` or `.gitignore` are exclude by this list 

```
*.swp
._*
.DS_Store
.git
.hg
.npmrc
.lock-wscript
.svn
.wafpickle-*
config.gypi
CVS
npm-debug.log
```

Comment (pedrocarrico) [inline .npmignore]: I think we should be OK with this current .npmignore, let's package it and adjust if needed.

Comment (cwhetung) [inline .npmignore]: :+1:

**Pull request merged**